db/sql/2_seattle.sql

```diff
@@ -71,13 +72,13 @@ SECURITY DEFINER
 SET search_path =public, pg_temp;
 
 
-CREATE OR REPLACE FUNCTION seattle_fuzzy_seattle_search_officer_by_name_p(full_name  VARCHAR(100))
+CREATE OR REPLACE FUNCTION seattle_fuzzy_search_officer_by_name_p(full_name_v  VARCHAR(100))
```

Comment:
Why `_v`?

Reply:
I had to add the `_v` to avoid collision with the `full_name` column name. I dunno if it was actually a problem but GoLand was recognizing it as a column name rather than a function input variable, so I changed it to appease the IDE gods.

Reply:
Gotcha!